### ISSUE TITLE
allow multiple custom buttons to update the data based on combined values

### ISSUE THREAD

**Issue by adithya8** [plotly/dash#1184]
After a lot of searching I wasn't able to find an existing solution for this problem I am facing.
I looked at the examples that use multiple buttons for 'restyling' graphs but not for 'updating'.

**Issue:**
To give a better picture of what I am dealing with, here's the what I want to plot.
I have 2 sets of related data A and B that needs to be displayed on prompt. Hence I was able to build that using an updatemenu with a dropdown. Now I want to have an option to change the scale of the values [Absolute scale, Log scale] for both A and B. Essentially both the data traces for A and B needs to transformed based on this option. I was not able to find how to do this just using update menus.

**Feature Request:**
I would like to have solution where we can have update menu buttons whose data attributes can be dependent on the active option on another button [hierarchical].

**Alternatives Considered:**
I considered writing the Scale update menu as a separate HTML button component and have a callback to update on click. I have also considered having the combination of all options [A-Absolute scale, B- Absolute scale, A- Log Scale, B- Log Scale]. 

Thank you for tech support!

**Comment by andreaslordos** [comment]
Struggling with the same issue right now. Did you happen to find a solution?

**Comment by adithya8** [comment]
@andreaslordos No, I just went with having buttons for all combinations of the options.

**Comment by AnnMarieW** [comment]
Thanks for the feature request.

`updatemenus` don't currently support dependent menus, where the options in one menu depend on the selection in another.

For this use case, rather than setting the `updatemenues` in the figure, it would be easier to use `dcc.Dropdown` (or other dash components) and use  Dash callbacks to update the figure. You could also ask on the [Plotly Community Forum](https://community.plotly.com/) for examples of how to do this in Dash.

Since this issue has been open for a while without much activity, it's best to close it for now.
